colossus/src/main/scala/colossus/core/WorkerManager.scala

```diff
@@ -24,7 +24,7 @@ import scala.util.{Failure, Success}
  * @param workerAgent WorkerRefs that this WorkerManager manages
  * @param ioSystem Containing IOSystem
  */
-private[colossus] class WorkerManager(workerAgent: Agent[IndexedSeq[WorkerRef]], ioSystem: IOSystem)
+private[colossus] class WorkerManager(workerAgent: Agent[IndexedSeq[WorkerRef]], ioSystem: IOSystem, workerFactory : WorkerFactory )
```

Comment:
workerFactory had to be injected now, since tests needed a test impl which allowed for Workers to be modeled by TestProbes.